docs/migrating.md

```diff
@@ -1 +1,84 @@
+| `SIGNALFX_ACCESS_TOKEN`                  | Splunk Observability Cloud access token for your organization. It enables sending telemetry directly to the Splunk Observability Cloud ingest endpoint.                                               |                                      | `SPLUNK_ACCESS_TOKEN`                                               |
+| `SIGNALFX_REALM`                         | Your Splunk Observability Cloud realm. To find your realm, open Splunk Observability Cloud, click Settings, then click on your username.                                                              | `none` (local collector)             | `SPLUNK_REALM`                                                      |
+| `SIGNALFX_ENDPOINT_URL`                  | The URL to where trace exporters send traces. Overrides `SIGNALFX_REALM` configuration for the traces ingestion endpoint.                                                                             | `http://localhost:9411/api/v2/spans` | `OTEL_EXPORTER_OTLP_ENDPOINT`/`OTEL_EXPORTER_OTLP_TRACES_ENDPOINT`  |
+| `SIGNALFX_METRICS_ENDPOINT_URL`          | The URL to where metric exporters send metrics. Overrides `SIGNALFX_REALM` configuration for the metrics ingestion endpoint.                                                                          | `http://localhost:9943/v2/datapoint` | `OTEL_EXPORTER_OTLP_ENDPOINT`/`OTEL_EXPORTER_OTLP_METRICS_ENDPOINT` |
+| `SIGNALFX_TRACE_PARTIAL_FLUSH_ENABLED`   | Enable to export traces that contain a minimum number of closed spans, as defined by `SIGNALFX_TRACE_PARTIAL_FLUSH_MIN_SPANS`.                                                                        | `false`                              | none                                                                |
+| `SIGNALFX_TRACE_PARTIAL_FLUSH_MIN_SPANS` | Minimum number of closed spans in a trace before it's exported. The default value is ``500``. Requires the value of the ``SIGNALFX_TRACE_PARTIAL_FLUSH_ENABLED`` environment variable to be ``true``. | `500`                                | none                                                                |
+| `SIGNALFX_TRACE_BUFFER_SIZE`             | The size of the trace exporter buffer, expressed as the number of traces.                                                                                                                             | `1000`                               | `OTEL_BSP_MAX_QUEUE_SIZE` (default: 2048)                           |
+
+## Trace propagation settings
+
+| Setting                | Description                                                                                                                                                   | Default   | Otel                                                                               |
```

Comment:
Otel? ;)

Reply:
fixed in https://github.com/signalfx/splunk-otel-dotnet/pull/124/commits/8b847f975a21de22ff2ec68b8a96cea358758dcb